Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamerBase.cpp

```diff
@@ -114,6 +114,13 @@
 #if ENABLE(ENCRYPTED_MEDIA) || ENABLE(ENCRYPTED_MEDIA_V2)
 #if ENABLE(ENCRYPTED_MEDIA_V2)
 #include "CDMPRSessionGStreamer.h"
+#if PLATFORM(WPE)
+#include <wpe/CDMPrivateEncKeyWPE.h>
+#include <wpe/CDMSessionEncKeyWPE.h>
+#include <Modules/encryptedmedia/CDM.h>
+#include <Modules/encryptedmedia/CDMPrivateClearKey.h>
```

Comment:
Are CDM.h and CDMPrivateClearKey.h inclusions needed here? I don't see these being used.